microcosm_elasticsearch/tests/test_registry.py

```diff
@@ -0,0 +1,28 @@
+"""
+Test registry management.
+
+"""
+from hamcrest import assert_that, equal_to, is_
+from microcosm.api import create_object_graph
+
+from microcosm_elasticsearch.registry import IndexRegistry
+
+
+def test_name_for():
```

Comment:
Would be cool to have a test around raising an exception for trying to create a duplicate index.